modules/nf-core/pigz/uncompress/main.nf

```diff
@@ -0,0 +1,48 @@
+process PIGZ_UNCOMPRESS {
+    label 'process_low'
+    //stageInMode 'copy' // this directive can be set in case the original input should be kept
```

Comment:
I think we need to set this in modules.config to keep the original input file(s)